### PR TITLE
Removes Geodesy dependency from Rhumb modules 2

### PR DESCRIPTION
**Pull request by stebogit**
Ref: https://github.com/Turfjs/turf/issues/971
https://github.com/Turfjs/turf/pull/973 sequel...

### CODE REVIEW

**Comment by DenisCarriere** [comment]
Woohoo! 🎉

**Comment by DenisCarriere** [comment]
@stebogit Resolved conflicts, 🤞 should pass...

I'll have a look at this again tomorrow, so far looks great! 👍

File: packages/turf-rhumb-bearing/index.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default function (start, end, options) {
  * const p2 = new LatLon(50.964, 1.853);
  * const d = p1.rhumbBearingTo(p2); // 116.7 m
  */
-function rhumbBearing(from, to) {
+function calculateRhumbBearing(from, to) {
```
Comment (DenisCarriere) [inline packages/turf-rhumb-bearing/index.js]: @stebogit I've been adding names to the export default function, in theory this shouldn't have any impact, however when bundling multiple modules sometimes the compile `dist/index.js` gets confused if another function has the same name as the main functions (`@turf/rhumbBearing`).

Have a look at the `dist/index.js` and you'll see how Rollup compiles the code.

Comment (stebogit) [reply packages/turf-rhumb-bearing/index.js]: 👍

**Pull request merged**